tools/chef/cookbooks/px_dev/recipes/setup.rb

```diff
@@ -29,5 +29,6 @@
 include_recipe 'px_dev::nodejs'
 include_recipe 'px_dev::php'
 include_recipe 'px_dev::python'
+include_recipe 'px_dev::packaging'
```

Comment:
Discussed in person, but would be good to move this to dev_w_extras only.

Reply:
Updated in 7a0c3eff7a1133b1dcd4395c9c8de4a9584ff4e7